satpy/readers/geocat.py

```diff
@@ -58,6 +58,12 @@
 class GEOCATFileHandler(NetCDF4FileHandler):
     """GEOCAT netCDF4 file handler."""
 
+    def __init__(self, filename, filename_info, filetype_info):
+        """Open and perform initial investigation of NetCDF file."""
+        super(GEOCATFileHandler, self).__init__(
+            filename, filename_info, filetype_info,
+            xarray_kwargs={"engine": "netcdf4", "decode_times": False})
```

Comment:
I guess the only change I'd like to consider is allowing the user to pass `xarray_kwargs` in a `**kwargs`. I think we had done something like `kwargs.setdefault("xarray_kwargs, {...})`.